framework/console/controllers/FixtureController.php

```diff
@@ -431,11 +437,18 @@ private function getFixturesConfig($fixtures)
         $config = [];
 
         foreach ($fixtures as $fixture) {
-            $isNamespaced = (strpos($fixture, '\\') !== false);
-            $fullClassName = $isNamespaced ? $fixture . 'Fixture' : $this->namespace . '\\' . $fixture . 'Fixture';
+            // $isNamespaced = (strpos($fixture, '\\') !== false);
```

Comment:
I think we can check for both `/` and `\` here or normalize everything to linux paths format before the check and the problem will be solved.